spec/board_maker_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+require 'board_maker'
+
+describe BoardMaker do
+  let(:maker) { BoardMaker.new }
+
+  context 'squares' do
```

Comment:
Use `describe` blocks for higher level categories - `context` is generally for when a certain condition is present. See [Betterspecs](http://betterspecs.org)